### DISCUSSION TITLE
Data storage supports Apache Doris

### DISCUSSION THREAD

**Discussion by Jengro777** [plausible/analytics#3075] [Anything Else]
Currently, all of Plausible's data is stored in ClickHouse. However, ClickHouse is difficult to maintain for small companies like ours. Can we consider supporting data storage in Apache Doris?

**Comment by ruslandoga** [comment]
👋 @Avey777 

I wonder what kind of problems you encounter while self-hosting ClickHouse?

**Reply by Jengro777** [reply]
Because we do not have enough manpower, we have just given up using ClickHouse to store any data.

**Reply by Jengro777** [reply]
A lot of our data is currently being stored on Apache Doris, a tool that has received unanimous recognition within our organization.
